app/controllers/api/v1/groups_controller.rb

```diff
@@ -2,7 +2,7 @@ class Api::V1::GroupsController < ApplicationController
   def index
     @groups = Group.all
 
-    render json: @groups
+    render json: @groups, :callback => params[:callback]
```

Comment:
My suggestion here is using `callback: params[:callback]` instead. It's only a sintax sugar, not a big deal 😄